app/assets/javascripts/jquery_overrides.js

```diff
@@ -37,10 +37,20 @@ $.ajaxSetup({
     }),
     "text script": logError(function (text) {
       if (text.match(/^{/)) {
+        console.debug('json payload');
         return jQuery.jsonPayload(text, function (text) {
           return text;
         });
+
+      } else if (text.match(/<!DOCTYPE html>/)) { // plain HTML payload fallback
+        console.debug('html payload');
+        document.open();
+        document.write(text);
```

Comment:
This way I can make all the `DoNav` links do a HXR request and if the target controller does not support the partial update the fallback will save the situation